card.css

```diff
@@ -0,0 +1,39 @@
+.product-card-vertical {
+  position: relative;
```

Comment:
If you have created card in your component library, then you can directly use card from there. Why you're creating separate card.css?